tests/llamapay/conftest.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+@pytest.fixture
+def safe():
+    return GreatApeSafe('ychad.eth')
+
+@pytest.fixture
+def dai(safe):
+    return safe.contract(r.tokens.dai)
+
+@pytest.fixture
+def usdc(safe):
+    return safe.contract(r.tokens.usdc)
+
+@pytest.fixture
+def payee():
+    return "0x8727FF875E070d73B678cCea85BF9FF01c05A785"
+
+@pytest.fixture
+def payee2():
+    return "0x40A63aDC56B32fdeF389FcB98571EdDC5e53daeD"
```

Comment:
Are these test accounts? Because instead of relying on the fact that these addresses currently have streams, it might be cleaner to generate new random accounts using `accounts.add()` here. Tests would then need to be set up in such a way that there is an actual stream to cancel of course.

Reply:
fixed in [e0a4e0e](https://github.com/Citadel-DAO/citadel-multisig/pull/23/commits/e0a4e0eb3b252383e6b4fea95efb6895a58b5576)